wp-content/plugins/core/src/Routes/Sample_Route.php

```diff
@@ -1,12 +1,5 @@
 <?php declare(strict_types=1);
 
-/**
```

Comment:
Why remove contextual comments?

Reply:
Same deal, plus it's in a spot above the namespace which none of other code has ever done.